Bump docs-theme-extras to v0.1.21, add pipefail to framework-tests, added descriptions (#36)
agentregistry-dev/website: content/docs/agents/deploy/kubernetes.md

---
title: Kubernetes
weight: 20

> description: "Deploy your agentregistry agent to a Kubernetes cluster."

Reviewer: ```suggestion
description: "Deploy an agent in your Kubernetes cluster."
```
